robot.py

```diff
@@ -70,6 +70,10 @@ def createObjects(self):
         self.lift_motor = ctre.WPI_TalonSRX(3)
         self.cube_switch = wpilib.DigitalInput(0)
 
+        self.lifter_motor = ctre.WPI_TalonSRX(3)
```

Comment:
Why do you define both `lift_motor` and `lifter_motor`? This breaks everything as they are on the same Talon ID...

Reply:
Also, this conflicts with the poorly-named `motor` variable you are injecting into the lifter component...